src/storage.cc

```diff
@@ -60,13 +60,13 @@ void Storage::InitBuyPolicyParameters() {
   }
 
   if (active_buying_frequency_type == "Fixed") {
-    active_dist_ = boost::shared_ptr<cyclus::FixedIntDist>(new cyclus::FixedIntDist(active_buying_val));
+    active_dist_ = cyclus::FixedIntDist::Ptr (new cyclus::FixedIntDist(active_buying_val));
   }
   else if (active_buying_frequency_type == "Uniform") {
     if ((active_buying_min == -1) || (active_buying_max == -1)) {
       throw cyclus::ValueError("Invalid active buying frequency range. Please provide both a min and max value.");
     }
-    active_dist_ = boost::shared_ptr<cyclus::UniformIntDist>(new cyclus::UniformIntDist(active_buying_min, active_buying_max));
+    active_dist_ = cyclus::UniformIntDist>(new cyclus::UniformIntDist(active_buying_min, active_buying_max));
```

Comment:
Looks like some typos here.... and below...

Reply:
oops

Reply:
Think I got them all now